### ISSUE TITLE
add to MELPA?

### ISSUE THREAD

**Issue by stardiviner** [abo-abo/pamparam#2]
As the title.

**Comment by abo-abo** [comment]
I don't mind putting it on MELPA eventually, once it's easier to use. Is it already usable in your opinion?

**Comment by stardiviner** [comment]
Yes, I found it from Youtube video, I saw the tutorial. It did have space for improvement. But release it can let some user use it earily like me.

**Comment by lchsk** [comment]
Hi
I'd also would like to see it on melpa. Cheers!

**Comment by abo-abo** [comment]
Already submitted, waiting approval.

**Comment by lchsk** [comment]
Great, thanks!

**Comment by MHohenberg** [comment]
Three years onward, and I would love to have this in MEPLA, if only to get an easy way to install this for emacs newbies.

**Comment by CaglanT** [comment]
@MHohenberg it is already on MELPA.
